### PR TITLE
Added non-root dockerfile

### PR DESCRIPTION
**Pull request by HackyPenguin**
Hi I have added a non-root dockerfile to ease in the deployment process

### CODE REVIEW

**Comment by cardoso** [comment]
Thanks HackyPenguin, I'll take a look soon

**Review by geekgonecrazy — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM python:3.6.5-alpine3.7
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh libffi-dev libffi pkgconf gcc g++ openssl openssl-dev shadow
+
+RUN useradd -u 9999 errbot
+
+Run  git clone https://github.com/cardoso/errbot-rocketchat
```
Comment (cardoso) [inline Dockerfile]: Can you make this RUN command uppercase?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+RUN chown -R errbot  errbot-rocketchat
+
+WORKDIR errbot-rocketchat
+
+Run python setup.py install
```
Comment (cardoso) [inline Dockerfile]: Can you make this RUN command uppercase?

**Review by cardoso — CHANGES_REQUESTED** [review]
Looking good @HackyPenguin ! Just some typing errors:

File: Dockerfile (hunk 1 above)
Comment (geekgonecrazy) [reply Dockerfile]: Totally missed that 😊

Comment (HackyPenguin) [reply Dockerfile]: Woops I also totally missed that ( workdays are to long)

**Comment by HackyPenguin** [comment]
Sorry about that  RUN is now upper

**Review by cardoso — APPROVED** [review]
Great! Thank you for contributing!

**Pull request merged**